### PR TITLE
Feature/alexis

### PR DESCRIPTION
**Pull request by lowbaj21**
# *Alexis View*

## :recycle: Current situation & Problem
*Previously no student for Alexis*

## :bulb: Proposed solution
*Added a new view for Alexis*

## :gear: Release Notes 
*Added profile photo and brief summary*

## :heavy_plus_sign: Additional Information
*N/A*

### Reviewer Nudging
*start at profile view*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [ ] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by rajpabari — APPROVED** [review]
Will tag Vishnu instead -- I should not be tagged for reviewing this PR

**Comment by lowbaj21** [comment]
> Will tag Vishnu instead -- I should not be tagged for reviewing this PR

Oops - sorry about that! I was just selecting from the suggested reviewers list and I guess not paying that much attention to the actual names.

File: CS342MemberViews/Sources/AlexisLowber/AlexisLowber.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
+//
+//  SwiftUIView.swift
+//  
+//
+//  Created by Alexis Lowber on 1/24/23.
+//
```
Comment (PSchmiedmayer) [inline CS342MemberViews/Sources/AlexisLowber/AlexisLowber.swift]: The REUSE specification requires all files to include licensing information. Please adjust your headers to follow the following format:
```suggestion
//
// This source file is part of the CS342 2023 Application project
//
// SPDX-FileCopyrightText: 2023 Stanford University
//
// SPDX-License-Identifier: MIT
//
```

**Review by PSchmiedmayer — APPROVED** [review]
@lowbaj21: Thank you for your PR.

I have reviewed your PR and have fixed a build issue that prevented me from building the application and viewing your view. I have also merged the latest main state in your branch to make merging the PR easy.
The only big missing element is a UI test to test your view.

Be sure that you `git pull` the latest state to your local version if you accept any of the suggestions here on GitHub and get the merge conflict to your local branch 🚀

Your current solution fits the minimum criteria for a submission but is limited in creativity. Be aware that some of the points given for the exercise value creativity and interactive elements. You are free to merge the PR and concentrate on your class project if you want to. I am also happy to review a second submission if you want to add some more interactive elements to your view.

The next steps to merge the PR would be:
1. Git pull the latest changes to your local copy
2. Address the REUSE error by adding the commonly used header to your file
3. Add a UI test for your view.
4. If you want, add some more interactive elements to your PR.

File: CS342MemberViews/Sources/AlexisLowber/AlexisLowber.swift (hunk 1 above)
Comment (lowbaj21) [reply CS342MemberViews/Sources/AlexisLowber/AlexisLowber.swift]: Thanks for the review. I'll update my branch and add a creative component!

**Comment by PSchmiedmayer** [comment]
@lowbaj21 A short reminder to improve your Assignment 1 submission. We would like to merge it in the next few days so you can concentrate on your project assignment. 

Feel free to tag me if you have any questions. I am also happy to help you with any merge conflicts that you might have to speed up the merge so that you can focus on the content of the exercise!

Feel free also to approach me after class today so we can take a look at the PR together and merge it directly if you have incorporated my feedback by then 🚀

**Comment by lowbaj21** [comment]
> 

Sounds good! Uploaded changes to the branch and will work on merging.

**Pull request merged**